include/fiction/algorithms/simulation/sidb/operational_domain.hpp

```diff
@@ -354,7 +355,7 @@ class operational_domain_impl
 
         // for each sample point in parallel
         std::for_each(FICTION_EXECUTION_POLICY_PAR_UNSEQ step_point_samples.cbegin(), step_point_samples.cend(),
-                      [this](const auto& sp) { is_step_point_operational(sp); });
+                      [this](const auto& sp) { is_step_point_operational(to_step_point(to_parameter_point(sp))); });
```

Comment:
@marcelwa This can be undone before merging. It is just there to illustrate the problem.